src/index.js

```diff
@@ -1,4 +1,4 @@
-const axios = require('axios');
+require("dotenv").config();
```

Comment:
💬 This library is great for loading contents from a `.env` file into `process.env`. It makes local development a lot easier.